package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cucumber-js-test",
+  "type": "commonjs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "cucumber-js",
+    "start": "npx ts-node src/index.ts",
```

Comment:
you using ts files, while real files are js

Reply:
Fixed